yang/ietf-ac-svc.yang

```diff
@@ -2066,45 +2066,51 @@ module ietf-ac-svc {
         description
           "Identification of the customer device that terminates the
            bearer.";
-        leaf device-id {
```

Comment:
In some cases, the identification of the site is not sufficient. We need to have the identification of the device(s) that will terminate an AC on the customer site. Think about the Network Functions attachment cases or peer ASBRs.